service_mesh/service_mesh_install/updating-ossm.adoc

```diff
@@ -4,7 +4,84 @@ include::modules/ossm-document-attributes.adoc[]
 :context: updating-ossm
 toc::[]
 
-If you are updating from {ProductName} 1.0 to 1.1, you must update the `ServiceMeshControlPlane` resource to update the control plane components to the new version. 
+
+== Manual updates required by version 1.1.4
+
+The fix for https://bugzilla.redhat.com/show_bug.cgi?id=1844254[CVE-2020-8663]`: envoy: Resource exhaustion when accepting too many connections` added a configurable limit on downstream connections. The configuration option for this limit must be configured to mitigate this vulnerability.
+
+This new configuration option is called `overload.global_downstream_max_connections`, and it is configurable as a proxy `runtime` setting.  Perform the following steps to configure limits at the ingress gateway.
```

Comment:
I see that we sometimes capitalize "Ingress Gateway"
https://docs.openshift.com/container-platform/4.4/service_mesh/service_mesh_user_guide/ossm-traffic-manage.html#ossm-routing-ingress_routing-traffic
Not sure which is correct, so just pointing it out.